doc/dev/installation.rst

```diff
@@ -132,6 +193,18 @@ For an existing copy of the repository without its submodules, they can also be
 
   git submodule update --init --depth=1
 
+All additional build and developer dependencies are managed via the repository's
+``requirements.txt`` and can be installed as follows once the repository is cloned:
+
+.. code-block:: console
+
+  pip install -r requirements.txt
+
+.. Note::
```

Comment:
```suggestion
.. note::
```

Reply:
There are other "Note::" in the documentation currently. Shall I change all of them to lower case?

Reply:
changed all of them to lower case